docs/pages/templates/marketplace/index.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Starter Kit'
+---
+
+import { Cards } from 'nextra-theme-docs'
+import { Card } from '../../../components/Card'
+
+# Starter kit
+
+The starter kit is fully fledged NFT marketplace running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
+
+This application is fork-able on this repository: https://github.com/liteflow-labs/starter-kit.
+
+<Cards num={2}>
+  <Card noPadding>![Home](/templates/marketplace/home.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/explore.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/detail.png)</Card>
+  <Card noPadding>![Home](/templates/marketplace/profile.png)</Card>
+</Cards>
+
+You can have a look at a full demo of this template here: https://demo.liteflow.com.
+
+## Features included
+
+<Cards>
+  <Card title="NFT Explorer" />
+  <Card title="NFT Minter" />
+  <Card title="Home page customization" />
+  <Card title="Search system" />
+  <Card title="NFT Trait filters" />
+  <Card title="Fixed price offer" />
+  <Card title="Open offer" />
+  <Card title="Offers in native token" />
+  <Card title="Offers in ERC20" />
+  <Card title="Partial fill for ERC1155" />
+  <Card title="User profile" />
+  <Card title="User wallet" />
+  <Card title="User dashboard" />
+  <Card title="User verification" />
+  <Card title="Report system" />
+  <Card title="In-app notification" />
+  <Card title="Email notification via webhook" />
+  <Card title="Wallet-to-wallet messaging" />
+  <Card title="Multiple wallet connection" />
+  <Card title="Multi-chain" />
+  <Card title="Multi-currency" />
+  <Card title="Multi-language" />
+</Cards>
+
```

Comment:
### Multi-Chain Compatibility
Our product seamlessly integrates with a variety of blockchain ecosystems to provide a versatile and expansive user experience. It is fully compatible with major chains including Ethereum, Polygon, Binance Smart Chain, and LightLink, as well as other EVM-compatible chains.